templates/core/terraform/network/network.tf

```diff
@@ -0,0 +1,53 @@
+resource "azurerm_virtual_network" "core" {
```

Comment:
We should have NSGs on the subnets.

Reply:
A new story #53 is created to define NSGs defaults.